modules/mod_xfer.c

```diff
@@ -2513,6 +2513,15 @@ MODRET xfer_retr(cmd_rec *cmd) {
     }
 
     len = transmit_data(cmd->pool, nbytes_sent, &curr_pos, lbuf, bufsz);
+
+	/*waiting 2 secons to check if the file is growing*/
+	sleep(2);
+
+	/*check the new size of the file*/
+	if (stat(dir, &st2) == 0){
+		session.xfer.file_size=st2.st_size;		
+	}
```

Comment:
It would greatly help if you _first_ described what issue/problem/behavior you are encountering, and why you feel this change best addresses that behavior.  Otherwise, I don't have a way to evaluate what this change is trying to accomplish, or why.